Profiler tools crash: "Cannot generate lazy proxy for service ...CollectorFormatter"

## Summary

Calling any Symfony profiler tool (e.g. `symfony-profiler-list`) fails with a
lazy-proxy generation error. The collector formatter services in the Symfony bridge are
declared `->lazy()`, and proxy generation fails inside Mate's standalone container even
though `symfony/var-exporter` is installed.

## Error

```
[ERROR] Error: Cannot generate lazy proxy for service
"Symfony\AI\Mate\Bridge\Symfony\Profiler\Service\Formatter\RequestCollectorFormatter".
```

## Environment

- Symfony AI Mate **0.10.0** (branch `mate-skills-install`, PR #2213)
- `symfony/ai-symfony-mate-extension` (same branch)
- PHP **8.3.30**
- `symfony/var-exporter` **v7.4.9** (present — so not a missing dependency)
- Host app: Symfony 7.4 webapp, profiler enabled, running under DDEV

## Steps to reproduce

1. Symfony 7.4 webapp with the profiler enabled (`composer require webapp`).
2. Install Mate + the Symfony extension from the branch.
3. `vendor/bin/mate discover`
4. Generate a profile: `curl http://<app>/` (any request).
5. Call a profiler tool:
   ```
   vendor/bin/mate mcp:tools:call symfony-profiler-list '{}'
   ```
6. → crashes with the error above.

## Cause

In `src/Bridge/Symfony/config/config.php`, the eight collector formatters are registered
`->lazy()`:

```php
$services->set(RequestCollectorFormatter::class)
    ->lazy()
    ->tag('ai_mate.profiler_collector_formatter');
// ... ExceptionCollectorFormatter, MailerCollectorFormatter, TranslationCollectorFormatter,
//     DoctrineCollectorFormatter, TimeCollectorFormatter, LoggerCollectorFormatter,
//     MemoryCollectorFormatter — all ->lazy()
```

Mate's standalone DI container appears not to be configured to generate these lazy proxies
at runtime.

## Workaround

Removing `->lazy()` from the eight formatter services makes the profiler tools work —
`symfony-profiler-list` then returns profiles normally. (These formatters are tiny, so the
lazy optimization buys little here.)

## Possible fixes

- Drop `->lazy()` from the formatter services, **or**
- configure the standalone container so lazy-proxy generation works (e.g. a proxy dumper /
  the right var-exporter wiring).

--- a/src/mate/src/Bridge/Symfony/config/config.php
+++ b/src/mate/src/Bridge/Symfony/config/config.php
@@ -61,35 +61,27 @@ return static function (ContainerConfigurator $configurator) {
 
         // Built-in collector formatters
         $services->set(RequestCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         $services->set(ExceptionCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         $services->set(MailerCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         $services->set(TranslationCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         $services->set(DoctrineCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         $services->set(TimeCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         $services->set(LoggerCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         $services->set(MemoryCollectorFormatter::class)
-            ->lazy()
             ->tag('ai_mate.profiler_collector_formatter');
 
         // MCP Capabilities

